Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml

```diff
@@ -0,0 +1,725 @@
+id: User added to local administrator group using a PowerShell command
+version: -1
+name: User added to local administrator group using a PowerShell command
+description: |-
+  This playbook is designed to handle the alert
+  'User added to local administrator group using a PowerShell command'
```

Comment:
you put here ' ' but in your first playbook you didn't put quotes
let's keep consistent without quotes